vendor/manifest

```diff
@@ -888,6 +888,14 @@
 			"branch": "v1"
 		},
 		{
+			"importpath": "gopkg.in/urfave/cli.v1",
+			"repository": "https://gopkg.in/urfave/cli.v1",
```

Comment:
Looks like we should be using "gopkg.in/urfave/cli.v1" as the import path in our own code to avoid the duplicate vendoring of this package.  If the NPE fix doesn't depend on updating this dependency, maybe the vendor update + import path change could be done in a separate PR?